modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20010,6 +20010,7 @@ this-message-does-not-have-file-attachments-in-the-recycle-bin=This message does
 this-name-is-already-in-use-try-another-one=This name is already in use. Try another one.
 this-number-refers-to-the-total-number-of-reads-since-the-content-was-published=This number refers to the total number of reads since the content was published.
 this-number-refers-to-the-total-number-of-views-since-the-content-was-published=This number refers to the total number of views since the content was published.
+this-object-includes-translatable-fields.-notification-terms-always-use-the-object-entrys-default-language=This object includes translatable fields. Notification Terms always use the entry's default language.
```

Comment:
You forgot:
=This object includes translatable fields. Notification Terms always use the **object** entry's default language.
Please, generate again.